.eslintrc.json

```diff
@@ -13,7 +13,7 @@
     "arrow-parens": ["error", "as-needed"],
     "comma-dangle": ["error", "never"],
     "indent": ["error", 2],
-    "linebreak-style": ["error", "unix"],
+    "linebreak-style": 0,
```

Comment:
could you tell me a little more about this.  I hadn't paid much attention to this recently.  I think we still want to keep a consistent line break approach ("\n"), but I'm open to discussion.  What does setting "linebreak-style" to zero achieve?  I **think** there's ways to do \n and not \r\n on Windows machines when you are using vscode...

Reply:
I'm not that strict about linebreak-style really, as I don't think it affects much except how things look on different editors/ides. Github check-in/check-out can also change linebreaks depending on your github settings, to allow for platform independent development. So, likebreak-style being inconsistent should be warning, rather than error.

Having said all that, the change I made, i.e. linebreak-style: 0, essentially disables linebreak-style checks. Don't know how to make it into a platform dependent warning, so that, if you're working on Windows, it warns if linebreaks are not Windows style, and if you're working on Unix, then it warns on non-Unix line breaks.

Do we even need to check linebreak styles for build as it doesn't affect output and code minifying removes them anyway.

Reply:
I think this would still lead to inconsistencies between platforms, which I'm hoping to avoid.  Would you be able to add an . editorconfig instead?